functions/isLeap.js

```diff
@@ -29,6 +29,21 @@ output: true
  * @return {boolean} whether the year is leap
  */
 
-// YOUR CODE HERE
-
+function isLeap(n){
+    var result; 
+    if (n/400){
+      result = true
+    }
+    else if(n/100){
+      result = false
+    }
+    else if(n/4){
+      result= true
+    }
+    else{
+      result= false
+    }
+    return result
+}
```

Comment:
Don't think this is correct, dear. You might want to go back and check this by running npm test isleap in your terminal